content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,34 +5,48 @@ weight = 20
 
 +++
 
+### What is Kubeflow?
+
 Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
 Operators can choose what is best for their users, there is no requirement to deploy every component.
-To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
+
+Learn more about Kubeflow on the [Introduction](/docs/started/introduction/) and [Architecture](/docs/started/architecture/) pages.
 
 ### How to install Kubeflow?
 
-There are two pathways to get up and running with Kubeflow, you may either:
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+There are two primary ways to install Kubeflow:
 
-1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
-2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
+1. [__Packaged Distributions__](#packaged-distributions-of-kubeflow)
+2. [__Raw Manifests__](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
```

Comment:
Do you need to remove for advance users as well ?
Since we removed warning from raw manifests installation section.

Reply:
We still kind of have a warning, it's just less forceful.

But either way, installing from the manifests is really for advanced users only, we don't want people new to Kubeflow/Kubernetes thinking that its a good place to start.

Reply:
From my point of view, we should even remove these warnings.
I think, users already understood this by reading the following:
"""
Packaged distributions are maintained by various organizations and typically aim to provide a **simplified** installation and management experience for Kubeflow.
"""

Would do others think @kubeflow/wg-manifests-leads @kubeflow/kubeflow-steering-committee ?

Reply:
I strongly think we need to keep a warning of some kind. 

The new warning from this PR is more specific that users need to understand Kubernetes, Istio and Kubeflow itself, but is overall less forceful (because if someone really is experienced, they shouldn't be afraid of it, but they should know what they're getting themselves into).

Kubeflow and it's dependencies are really one of the most complicated things people regularly deploy on Kubernetes.

Reply:
I agree with Andrew here. Let's rather work with the smallest denominator and refine later on if needed. Then we can merge this PR without the warning and bring any controversial ideas to the KSC agenda.

Reply:
Just so we are clear, the existing website says `(for advanced users)`, and this PR just removes the word `"for"` so it becomes `(advanced users)`, so its not really a significant change.

I can re-add the "for" if you like, lol.

If we want to change how we present the "raw manifests" to be a non-advanced option, we should do that in a separate PR, after a lot of community discussion.

Reply:
Yes of course, if is really just removing "for"